traffic_ops/traffic_ops_golang/login/register.go

```diff
@@ -208,7 +208,7 @@ func RegisterUser(w http.ResponseWriter, r *http.Request) {
 
 	var role string
 	var tenant string
-	user, exists, err := dbhelpers.GetUserByEmail(req.Email.Address.Address, inf.Tx.Tx)
+	user, exists, err := dbhelpers.GetUserByEmail(inf.Tx, req.Email.Address.Address)
```

Comment:
I don't think you meant to change this either

Reply:
See [comment](https://github.com/apache/trafficcontrol/pull/4633#issuecomment-617996513), fixed in 003af41431.